projects/1.20/assets/netherific/nourished_nether/lang/zh_cn.json

```
@@ -0,0 +1,151 @@
+  "block.nourished_nether.chiseled_soul_quartz": "浮雕灵魂石英",
+  "block.nourished_nether.chiseled_warped_nether_bricks": "浮雕诡异下界砖",
+  "block.nourished_nether.cracked_basalt_bricks": "裂纹玄武岩砖",
+  "block.nourished_nether.cracked_obsidian_bricks": "裂纹黑曜石砖",
+  "block.nourished_nether.cracked_quartz_bricks": "裂纹石英砖",
+  "block.nourished_nether.cracked_red_nether_bricks": "裂纹红色下界砖",
+  "block.nourished_nether.cracked_sludge_bricks": "裂纹泥浆砖",
+  "block.nourished_nether.cracked_smooth_basalt_bricks": "裂纹平滑玄武岩砖",
+  "block.nourished_nether.cracked_soul_quartz_bricks": "裂纹灵魂石英砖",
+  "block.nourished_nether.cracked_warped_bricks": "裂纹诡异下界砖",
+  "block.nourished_nether.ectoplasm_block": "灵质块",
+  "block.nourished_nether.ectoplasm_block_active": "灵质块",
+  "block.nourished_nether.ghoulflower": "幽灵花",
+  "block.nourished_nether.gilded_blackstone_brick_slab": "镶金黑石砖台阶",
+  "block.nourished_nether.gilded_blackstone_brick_stairs": "镶金黑石砖楼梯",
+  "block.nourished_nether.gilded_blackstone_brick_wall": "镶金黑石砖墙",
+  "block.nourished_nether.gilded_blackstone_bricks": "镶金黑石砖",
+  "block.nourished_nether.glowing_obsidian": "发光黑曜石",
+  "block.nourished_nether.hoglin_hide_block": "疣猪皮块",
+  "block.nourished_nether.necronium_block": "死灵块",
+  "block.nourished_nether.nether_brick_fence_gate": "下界砖栅栏门",
+  "block.nourished_nether.nether_reactor_core": "下界反应堆核心",
+  "block.nourished_nether.nether_reactor_rod": "下界反应堆棒",
+  "block.nourished_nether.netherite_scrap_block": "下界合金残片块",
+  "block.nourished_nether.netherrack_slab": "下界岩台阶",
+  "block.nourished_nether.netherrack_stairs": "下界岩楼梯",
+  "block.nourished_nether.netherrack_wall": "下界岩墙",
+  "block.nourished_nether.obsidian_brick_slab": "黑曜石砖台阶",
+  "block.nourished_nether.obsidian_brick_stairs": "黑曜石砖楼梯",
+  "block.nourished_nether.obsidian_brick_wall": "黑曜石砖墙",
+  "block.nourished_nether.obsidian_bricks": "黑曜石砖",
+  "block.nourished_nether.obsidian_slab": "黑曜石台阶",
+  "block.nourished_nether.obsidian_stairs": "黑曜石楼梯",
+  "block.nourished_nether.obsidian_wall": "黑曜石墙",
+  "block.nourished_nether.polished_basalt_slab": "磨制玄武岩台阶",
+  "block.nourished_nether.polished_basalt_stairs": "磨制玄武岩楼梯",
+  "block.nourished_nether.polished_basalt_wall": "磨制玄武岩墙",
+  "block.nourished_nether.quartz_brick_slab": "石英砖台阶",
+  "block.nourished_nether.quartz_brick_stairs": "石英砖楼梯",
+  "block.nourished_nether.quartz_brick_wall": "石英砖墙",
+  "block.nourished_nether.quartz_wall": "石英墙",
+  "block.nourished_nether.red_nether_brick_fence": "红色下界砖栅栏",
+  "block.nourished_nether.red_nether_brick_fence_gate": "红色下界砖栅栏门",
+  "block.nourished_nether.rich_soul_sludge": "富灵魂泥浆",
+  "block.nourished_nether.sludge_brick_fence": "泥浆砖栅栏",
+  "block.nourished_nether.sludge_brick_fence_gate": "泥浆砖栅栏门",
+  "block.nourished_nether.sludge_brick_slab": "泥浆砖台阶",
+  "block.nourished_nether.sludge_brick_stairs": "泥浆砖楼梯",
+  "block.nourished_nether.sludge_brick_wall": "泥浆砖墙",
+  "block.nourished_nether.sludge_bricks": "泥浆砖",
```

```suggestion
  "block.nourished_nether.sludge_bricks": "泥浆砖块",
```
模组中有对应砖物品，遵从原版`红砖`和`红砖块`、`下界砖`和`下界砖块`格式
`红色下界砖`和`蓝色下界砖`也是一样的要求